real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -101,10 +102,40 @@ public static void loadConfigFile() {
             setWayPointSourceLogFile(configJson.get("WAYPOINT_SOURCE_LOG_FILE").getAsString());
             setPlayBackSourceFile(configJson.get("PLAY_BACK_SOURCE_FILE").getAsString());
 
-
         } catch (UnsupportedEncodingException | FileNotFoundException e) {
             new RobobuggyLogicNotification("could not load Part of configFile" + e.toString(), RobobuggyMessageLevel.WARNING);
         }
+        getAliceLibraryVersion();
+    }
+
+    /**
+     * @return the current library version
+     */
+    public static String getAliceLibraryVersion() {
+        if (!aliceLibraryVersion.equals("")) {
+            return aliceLibraryVersion;
+        }
+        else {
+            try {
+                String gitCommand = "git show";
```

Comment:
Git show can potentially get really messy.  Why not use what we use in fingerprint.py?